platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/schema_metacard.xml

```diff
@@ -603,43 +613,82 @@
           Longer patterns will be matched first.  if equal size patterns
           both match, the first appearing in the schema will be used.  -->
 
-     <dynamicField name="*_int" type="tint" indexed="true" stored="true"/>
-   
-   <dynamicField name="*_lng" type="tlong" indexed="true" stored="true"/>
-   
-   <dynamicField name="*_shr" type="tint" indexed="true" stored="true"/>
-   
-   <dynamicField name="*_flt" type="tfloat" indexed="true" stored="true"/>
-   
-   <dynamicField name="*_dbl" type="tdouble" indexed="true" stored="true"/>
-
-   <dynamicField name="*_tdt" type="tdate" indexed="true" stored="true"/>
-   
-   <dynamicField name="*_bln" type="boolean" indexed="true" stored="true"/>
-   
-   <dynamicField name="*_geo" type="string" indexed="false" stored="true"/>
-
-   <dynamicField name="*_geo_index" type="geohash" indexed="true" stored="false"/>
-   
-   <dynamicField name="*_txt" type="string" indexed="true" stored="true"/>
-   
-   <dynamicField name="*_xml" type="string" indexed="false" stored="true"/>
-   
-   <dynamicField name="*_txt_tokenized" type="text_general" indexed="true" stored="false"/>
-   
+   <dynamicField name="*_int" type="tint" indexed="true" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_lng" type="tlong" indexed="true" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_shr" type="tint" indexed="true" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_flt" type="tfloat" indexed="true" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_dbl" type="tdouble" indexed="true" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_tdt" type="tdate" indexed="true" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_bln" type="boolean" indexed="true" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_geo" type="string" indexed="false" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_geo_index" type="geohash" indexed="true" stored="false" multiValued="true"/>
+
+   <dynamicField name="*_txt" type="string" indexed="true" stored="true" multiValued="true"/>
+
+   <dynamicField name="*_txt_set" type="string" indexed="true" stored="true" multiValued="true"/>
```

Comment:
`_txt_set` should be removed and `PersistenceStoreImpl` should just use the new multivalued `_txt`.